containers/Reports/ReportCreateMessage.styles.js

```diff
@@ -6,24 +6,9 @@ import { SPACINGS } from '../../theme/spacings';
 export const styles = StyleSheet.create({
   container: {
     backgroundColor: COLORS.paper.main,
+    paddingTop: SPACINGS.multiplyBy(.25),
```

Comment:
somehow it was needed in `ideaCreate` and I didn't find out why it behaves different there. I just assumed it's because of some CSS stuff I don't get (padding of another element or dunno). If you know where to change I'm all for it.

Reply:
I moved the Header out of the KeyboardAvoidingView (as it apparently comes with padding/margin already) in `Idea.js` and set the padding on the Header directly now.